geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/index/BinLineStringTest.scala

```diff
@@ -36,14 +36,14 @@ class BinLineStringTest extends Specification with TestWithDataStore {
       val sf = new ScalaSimpleFeature(s"$i", sft)
       val geom = s"LINESTRING(40 6$i, 40.1 6$i, 40.2 6$i, 40.3 6$i)"
       val dates = new java.util.ArrayList[Date]
-      (0 to 4).map(mm => java.util.Date.from(java.time.LocalDateTime.parse(s"2010-05-07T0$i:0$mm:00.000Z", GeoToolsDateFormat).toInstant(java.time.ZoneOffset.UTC))).foreach(dates.add)
+      (0 until 4).map(mm => java.util.Date.from(java.time.LocalDateTime.parse(s"2010-05-07T0$i:0$mm:00.000Z", GeoToolsDateFormat).toInstant(java.time.ZoneOffset.UTC))).foreach(dates.add)
```

Comment:
this was an oversight in the original test i think - the number of dates is supposed to match the number of points in the line string. I started logging that case in the iterator, so the test started spitting out a bunch of warnings.